### ISSUE TITLE
Minify works but floods console with errors [Yosemite ST3]

### ISSUE THREAD

**Issue by antenando** [tssajo/Minify#34]
I can minify just fine but the following error floods my ST3 console:

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 441, in is_enabled_
    ret = self.is_enabled()
  File "/Users/lbueno/Library/Application Support/Sublime Text 3/Packages/Minify/Minify.py", line 79, in is_enabled
    return bool((type(filename).__name__ in ('str', 'unicode') and (re.search(r'\.(?:css|js|json|html?|svg)$', filename)) or (re.search(r'(\.[^\.]+)$', filename) and re.search(r'/(?:CSS|JavaScript|JSON|HTML)\.tmLanguage$', self.view.settings().get('syntax')))))
  File "./re.py", line 161, in search
TypeError: expected string or buffer
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 439, in is_enabled_
    ret = self.is_enabled()
  File "/Users/lbueno/Library/Application Support/Sublime Text 3/Packages/Minify/Minify.py", line 79, in is_enabled
    return bool((type(filename).__name__ in ('str', 'unicode') and (re.search(r'\.(?:css|js|json|html?|svg)$', filename)) or (re.search(r'(\.[^\.]+)$', filename) and re.search(r'/(?:CSS|JavaScript|JSON|HTML)\.tmLanguage$', self.view.settings().get('syntax')))))
  File "./re.py", line 161, in search
TypeError: expected string or buffer

During handling of the above exception, another exception occurred:
```

Any clues?

Cheers, the plugin is awesome.

**Comment by tssajo** [comment]
Thanks for reporting this. I'll try to fix this sometime next week. I am
out of town this week.
Zoltan

**Comment by tssajo** [comment]
This is fixed in `Minify` version 1.2.2

**Comment by antenando** [comment]
![vcm9ww](https://cloud.githubusercontent.com/assets/61152/9159562/0cc3ecda-3f01-11e5-9e5f-d66fff44bf5f.jpg)

*and thanks for the kind mention* ;)
